src/bin/lttng-sessiond/main.c

```diff
@@ -3495,10 +3620,11 @@ static int process_client_msg(struct command_ctx *cmd_ctx, int sock,
 			ret = lttcomm_recv_unix_sock(sock, filter_expression,
 				expression_len);
 			if (ret <= 0) {
-				DBG("Nothing recv() from client car len data... continuing");
+				DBG("Nothing recv() from client var len data... continuing");
```

Comment:
Thanks for fixing this! Receiving a _car length_'s worth of data ;-)

Reply:
ahah :car: